### ISSUE TITLE
The latest Swift version is 4.1 but SDK is still compiled by Swift 3.2

### ISSUE THREAD

**Issue by silverautoJosh** [AppLozic/Applozic-iOS-Chat-Samples#19]
Hello, I have used CocoaPods to install the Applozic SDK in our iOS project. But the Applozic SDK was compiled by Swift 3.2 which is not compatible with the latest Swift version 4.1. Do you have plan to upgrade the Swift version for your SDK? Thanks

**Comment by mukeshthawani** [comment]
Yes, we will release a new version of our SDK with Swift 4.1 support.
